src/repositorios/collection.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const Sequelize = require("sequelize");
+
+const CollectionModel = require("../models/CollectionModel");
+
+
+const db = require("../database");
+
+module.exports = {
+
+    async create_collection(collection){
+        return CollectionModel.create(collection);
+    },
+
+    async delete_collection(id_collection){
```

Comment:
We want to mark for deletion instead of delete. Need to add a new field `delete`.